Parser/Client/AbstractClientParser.php

```diff
@@ -82,6 +82,31 @@ public static function getAvailableClients(): array
                 continue;
             }
 
+            if ('Microsoft Office $1' === $regex['name']) {
+                $clients = [
+                    'Microsoft Office Access', 'Microsoft Office Excel', 'Microsoft Office OneDrive for Business',
+                    'Microsoft Office OneNote', 'Microsoft Office PowerPoint', 'Microsoft Office Project',
+                    'Microsoft Office Publisher', 'Microsoft Office Visio', 'Microsoft Office Word',
+                ];
+
+                $regex['name'] = \implode(', ', $clients);
```

Comment:
I think we should do it like this:
```php
foreach($clients as $client) {
   $names[] = $client;
}
continue;
```
this will not cause confusion when I want to check if there is such a client in the array
```php
in_array('Microsoft Office Publisher', MobileAppParser::getAvailableClients())
```